### PR TITLE
AddRouteAnnotationRector: Some bugfixes

### PR DESCRIPTION
**Pull request by malteschlueter**
* Fix issue that a `null` default parameter was a empty string as annotation
* Fix issue that boolean default parameter was a numeric string as annotation
* Fix issue that host was not quoted

https://github.com/rectorphp/rector-symfony/issues/242

### CODE REVIEW

File: tests/Rector/ClassMethod/AddRouteAnnotationRector/Fixture/fixture_missing_route.php.inc [hunk 1]
```diff
@@ -26,7 +26,7 @@ use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 class AppController extends Controller
 {
     /**
-     * @\Symfony\Component\Routing\Annotation\Route(path="/all/{foo}/{page}", name="all", defaults={"foo" = "foo123", "page" = 1}, host=m.example.com, schemes={"http", "https"}, methods={"GET", "POST"}, condition="context.getMethod() in ['GET', 'HEAD'] and request.headers.get('User-Agent') matches '/firefox/i'", requirements={"page" = "\d+"}, options={"compiler_class" = "App\Routing\Utf8RouteCompiler", "expose" = "1", "foo1" = 5, "foo2" = "bar"})
```
Comment (TomasVotruba) [inline tests/Rector/ClassMethod/AddRouteAnnotationRector/Fixture/fixture_missing_route.php.inc:29]: Could you add this as a new test fixture? The null case might repeat for other annotations too.

**Review by TomasVotruba — COMMENTED** [review]
Thanks 👍 I've left a comment.

File: tests/Rector/ClassMethod/AddRouteAnnotationRector/Fixture/fixture_missing_route.php.inc (hunk 1 above)
Comment (malteschlueter) [reply tests/Rector/ClassMethod/AddRouteAnnotationRector/Fixture/fixture_missing_route.php.inc:29]: Do you mean like this? https://github.com/rectorphp/rector-symfony/pull/246/files#diff-70343deb30a79daad564ef30ec87ee6db89ea53dc10eb30e988b88b3985636bb

Comment (TomasVotruba) [reply tests/Rector/ClassMethod/AddRouteAnnotationRector/Fixture/fixture_missing_route.php.inc:29]: Yes, exactly :+1: 

Thanks

**Comment by TomasVotruba** [comment]
Thank you :clap:
